syntax = "proto3";

package endpoints;

message TerminateRequest {
}

message TerminateResponse {
}
